hg-fast-export.py

```diff
@@ -515,6 +576,8 @@ def bail(parser,opt):
       help="Additional search path for plugins ")
   parser.add_option("--plugin", action="append", type="string", dest="plugins",
       help="Add a plugin with the given init string <name=init>")
+  parser.add_option("--subrepo-map", type="string", dest="subrepo_map",
+      help="Provide a mapping file between the subrepository name and the submodule name, see README-SUBMODULES.md")
```

Comment:
Drop the `, see README-SUBMODULES.md` part, a user running a packaged version won't know where it is located.

Reply:
Will do!